rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -535,3 +561,56 @@ SettingsDialog::SettingsDialog(wxWindow* parent)
 		config.write(out.c_str(), out.size());
 	}
 }
+
+void SettingsDialog::OnModuleListItemToggled(wxCommandEvent &event)
+{
+	lle_module_list[fmt::ToUTF8(event.GetString())] = chbox_list_core_lle->IsChecked(event.GetSelection());
+}
+
+void SettingsDialog::OnSearchBoxTextChanged(wxCommandEvent &event)
+{
+	if (event.GetString().IsEmpty())
+	{
+		for (auto& i : lle_module_list)
+		{
+			if (i.second)
+			{
+				chbox_list_core_lle->Check(chbox_list_core_lle->Append(i.first));
+			}
+		}
+
+		for (auto& i : lle_module_list)
+		{
+			if (!i.second)
```

Comment:
the idea behind this is to show the selected modules always on top, with just one loop this does not work as expected